shared/templates/sysctl/bash.template

```diff
@@ -6,7 +6,7 @@
 
 # Comment out any occurrences of {{{ SYSCTLVAR }}} from /etc/sysctl.d/*.conf files
 {{% if product in [ "sle12", "sle15"] %}}
-for f in /run/sysctl.d/*.conf /etc/sysctl.d/*.conf /usr/local/lib/sysctl.d/*.conf /usr/lib/sysctl.d/*.conf /lib/sysctl.d/*.conf; do
+for f in /etc/sysctl.d/*.conf /run/sysctl.d/*.conf /usr/local/lib/sysctl.d/*.conf /lib/sysctl.d/*.conf; do
```

Comment:
`/lib` directory is a symlink to `/usr/lib` at least for Fedora/RHEL-alike distributions if I'm not mistaken. So this probably needs to be removed as well.

Reply:
Surprisingly, Debian has issues with this as of 2022, but this is indeed the case for everybody else for some time (apparently including Ubuntu): https://lwn.net/Articles/890219/

Reply:
This be potentially merged with the elif statement as they will become very similar in case /lib and /usr/lib are dropped from both cases.